src/renderer/components/Dids/index.vue

```diff
@@ -33,11 +32,58 @@ import DidList from './DidList.vue';
 import message_bus from '../../message_bus.js';
 import share from '../../share.js';
 
+export const shared = {
+  data: {
+    dids: [],
+    public_did: ''
+  },
+  listeners: {
+    'did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/admin-dids/1.0/list-dids':
+    (share, msg) => share.dids = msg.result,
```

Comment:
I believe we can indent this without making the parser angry. Some indentation might make this a little more clear.